Universal/usbw.md

```diff
@@ -0,0 +1,15 @@
+# Fixing USB Wake (SSDT-USBW)
+
+* [Fixing USB Wake (SSDT-USBW)](#fixing-usb-wake-ssdt-usbw)
+  * [What this SSDT does](#what-this-ssdt-does)
+  * [Methods to make this SSDT](#methods-to-make-this-ssdt)
+
+## What this SSDT does
+
+SSDT-USBW is a companion to the [USBWakeFixup kext](https://github.com/osy/USBWakeFixup), which addresses USB wake issues on certain Intel platforms. Do not use this SSDT unless you have the USBWakeFixup kext.
```

Comment:
This seems like a last resort when adding `acpi-wake-type` doesn't work. Could you maybe explain a bit more about what is going on here and other prerequisites?